packages/realm/RealmJS.podspec

```diff
@@ -35,13 +39,16 @@ Pod::Spec.new do |s|
   s.homepage               = package['homepage']
   s.platform               = :ios, '9.0'
 
-  # The source field is a required field in the podspec, but it is not ment to be used.
-  # This is because the Podspec is not ment to be published into a CocoaPod repository, instead React Native uses a :path style dependency when adding this to the users projects Podfile.
+  # The source field is a required field in the podspec, but it is not meant to be used.
+  # This is because the Podspec is not meant to be published into a CocoaPod repository, instead React Native uses a :path style dependency when adding this to the users projects Podfile.
   # @see https://guides.cocoapods.org/using/the-podfile.html#using-the-files-from-a-folder-local-to-the-machine
   # @see https://github.com/react-native-community/cli/blob/master/docs/autolinking.md#platform-ios
   s.source                 = { :http => 'https://github.com/realm/realm-js/blob/main/CONTRIBUTING.md#how-to-debug-react-native-podspec' }
 
-  s.source_files           = 'react-native/ios/RealmReact/*.mm'
+  s.source_files           = 'react-native/ios/RealmReact/*.mm',
+                             'binding/jsi/*.cpp',
+                             'binding/ios/platform.mm'
+
   s.public_header_files    = 'react-native/ios/RealmReact/*.h'
 
   s.frameworks             = uses_frameworks ? ['React'] : []
```

Comment:
If we can avoid '"$(PODS_ROOT)/Headers/Public/React-Core/"' in `'HEADER_SEARCH_PATHS'`, I'm left wondering if we could avoid this as well?

Reply:
Are you suggesting removing the `uses_frameworks` check?